Libraries/LibJS/Tests/Array.prototype-generic-functions.js

```diff
@@ -0,0 +1,47 @@
+load("test-common.js");
+
+try {
+    const o = { length: 5, 0: "foo", 1: "bar", 3: "baz" };
+
+    ["every"].forEach(name => {
```

Comment:
I'is there a reason for "every" to be away from "filter", forEach", etc..., or is it leftover from debugging ?

Reply:
It's on purpose, the latter have `return false` in the callback and `every` has `return true`, so it doesn't abort after the first item and visits all of them for the sake of testing.

Reply:
Oh, didn't see that !